### PR TITLE
(FIX) Firepad: Opening a file takes more than 5 seconds sometimes

### PR DESCRIPTION
**Pull request by mohanpierce99**

### CODE REVIEW

**Comment by Progyan1997** [comment]
@mohanpierce99 Can you check Travis failures?

File: examples/monaco.html [hunk 1]
```diff
@@ -51,12 +52,11 @@
...
           });
-          var firepad = Firepad.fromMonaco(firepadRef, editor, {
-            defaultText: '// typescript Editing with Firepad!\nfunction go() {\n  var message = "Hello, world.";\n  console.log(message);\n}'
-          });
+          editor.setValue('Hello world')
+          var firepad = Firepad.fromMonaco(firepadRef, editor);
           window.firepad = firepad;
           window.editor = editor;
-
+
```
Comment (Progyan1997) [inline examples/monaco.html:59]:
```suggestion

```

File: lib/firepad.js [hunk 2]
```diff
@@ -107,6 +103,9 @@ firepad.Firepad = (function(global) {
     } else {
       this.editorAdapter_ = new MonacoAdapter(this.monaco_);
     }
+    this.editor_.adapter = this.editorAdapter_
+    this.firebaseAdapter_ = new FirebaseAdapter(ref, userId, userColor,this.editor_);
```
Comment (Progyan1997) [inline lib/firepad.js:107]: I thought we decided to do it event-driven?

Comment (mohanpierce99) [reply lib/firepad.js:107]: Added now

**Pull request closed without merging**